drivers/sensor/lm75/lm75.c

```diff
@@ -66,11 +66,12 @@ static int lm75_sample_fetch(const struct device *dev,
 	const struct lm75_config *cfg = dev->config;
 	enum pm_device_state pm_state;
 	int ret;
+	int pm_ret;
```

Comment:
why an extra variable when you already have `ret`?

Reply:
@gmarull because using only ret, I couldnt print the error from the PM subsystem and keep the main error info to return it in the end.

Reply:
> @gmarull because using only ret, I couldnt print the error from the PM subsystem and keep the main error info to return it in the end.

Not sure how is this relevant, if you have <0 you'll have already returned.